sdk/sdk.gradle

```diff
@@ -1,4 +1,4 @@
-apply from: "./protobuf-lite.gradle"
+apply from: "./protobuf.gradle"
```

Comment:
Example won't work without this change.

Reply:
Don't change the protobuf dependency. The `sdk` artifact should use `protobuf-lite` and the `sdk-full` should use `protobuf`